### DISCUSSION TITLE
Nothing plotted due to incorrect parse of the results

### DISCUSSION THREAD

**Discussion by felixmondelo** [marcusolsson/grafana-gantt-panel#36] [Q&A] [answered]
Hi,

Sometimes nothing is plotted due to an incorrect parse of the query result, it mixes in the header data from other columns:

![image](https://user-images.githubusercontent.com/14011841/121222347-c28b1600-c886-11eb-965b-b93f93e4fb80.png)

Usually, it works fine and plots the results and the results data are on each column:

![image](https://user-images.githubusercontent.com/14011841/121222323-bdc66200-c886-11eb-9958-286ef19871df.png)

How can I fix this?

**Comment by marcusolsson** [comment]
Thanks for sharing! Though I'm not sure I understand. In the first example, it looks like you're using a Time series format, and in the second example you're using Table format? This doesn't look like an issue with the Gantt panel, but with your query/data source? Could you provide more details? Ideally so that I can reproduce this myself.

**Comment by felixmondelo** [comment]
The queries are against Azure Data Explorer and both are the same query but with several time ranges. I don't know why sometimes with some filter, I receive:

![image](https://user-images.githubusercontent.com/14011841/121651175-a233a500-ca9a-11eb-8aff-446a466f18f8.png)

When I inspect the Data, I saw that when the Gant fails is because the data is formatted as:

![image](https://user-images.githubusercontent.com/14011841/121651489-fa6aa700-ca9a-11eb-984d-5a82961919db.png)

And when works, the data is formatted:

![image](https://user-images.githubusercontent.com/14011841/121651532-05bdd280-ca9b-11eb-83f5-c9baa2f61da9.png)

If I launch both queries on ADX the structure is equal:

> This fails:

![image](https://user-images.githubusercontent.com/14011841/121651690-300f9000-ca9b-11eb-92a7-7d4ff4aad5a8.png)

> This works:

![image](https://user-images.githubusercontent.com/14011841/121651822-52a1a900-ca9b-11eb-91cb-0266bd179c7f.png)

How can I help you?

**Comment by marcusolsson** [comment]
That does sound weird. The Gantt panel only works if you've set **Format as** to **Table**. If you're getting time series data even when you've set it to return Table data, it sounds like it could be an issue in ADX. Maybe @joshhunt can weigh in here?

**Comment by felixmondelo** [comment, marked as answer]
Yes, I think was that format the query to **Table** ... It seems now works, I continue testing ... Thank you so much!
